src/community/propose-a-component-or-pattern/index.md.njk

```diff
@@ -15,7 +15,11 @@ Follow the steps to propose a component or pattern for the Design System.
 
 ## 1. Check the list of discussions on GitHub
 
+<<<<<<< HEAD
```

Comment:
Someone's accidentally committed a merge conflict. This'll need to get fixed before this goes out. There's nothing special about this, just a case of picking what content you want to go here and getting rid of the extra weird git stuff.

Reply:
Probably me, as you can tell from the commit history. 

I tried to squash some commits in GitHub Desktop and when I pulled it these conflicts got flagged. I can go in again to pick and clean-up tomorrow, may need help to do the squashing later on.